web/src/utils/state/index.ts

```diff
@@ -3,6 +3,9 @@ import { atomWithStorage, createJSONStorage } from 'jotai/utils';
 import { Mode, ThemeOptions, TimeAverages, ToggleOptions } from '../constants';
 import atomWithCustomStorage from './atomWithCustomStorage';
 
+export const loadingMapAtom = atom(true);
+loadingMapAtom.debugLabel = 'loadingMap';
+
```

Comment:
I assume we are going to use Babel later in this version as well.
So we might want to switch from custom debugLabels to using https://github.com/pmndrs/jotai/blob/main/docs/api/babel.mdx#plugin-debug-label when we do.

```suggestion
export const loadingMapAtom = atom(true);
// TODO: Switch to Babel debuging labels when Babel is added.
loadingMapAtom.debugLabel = 'loadingMap';

```

Reply:
I guess it's a preformance/size thing?
Just keeping them as numbers would allow for substantial size savings in barebones and larger apps alike.

Reply:
Yeah, I would just imagine that the `useAtomsDevtools` should be able to do something smart on process.env.DEV, but I assume that's not possible since they didn't do it 😄